exporters/pelorus/timeutil.py

```diff
@@ -5,10 +5,13 @@
 will _always_ produce timezone-aware objects,
 which are necessary for correctness with `astimezone(tz)`, `timestamp()`, and other methods.
 """
-from datetime import datetime, timezone
+from datetime import datetime, timedelta, timezone
 
 _ISO_ZULU_FMT = "%Y-%m-%dT%H:%M:%SZ"
 
+# Time after which metrics for the deploytime exporter will not be accepted
```

Comment:
I believe it is not necessary this comment

but if you want to keep it, this will apply for everything, right? Not just deploytime exporter